modules/cloud-source-paladin-cloud.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-paladin-cloud.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="cloud-source-paladin-cloud_{context}"]
+= Configuring Paladin Cloud integration
+
+To discover cluster assets from Paladin Cloud, create a new integration in {product-title}.
+
+.Prerequisites
+* A Paladin Cloud account.
+* A Paladin Cloud API token.
+
+.Procedure
+. On the {product-title-short} portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Cloud source integrations* section and select *Paladin Cloud*.
+. Click *New Integration* (*`add`* icon).
```

Comment:
I'm not super familiar with UX terminology, so my feedback is limited here, but I think unless "add" appears as a phrase in a file or code output, basically anything you'd see on a CLI, the literals shouldn't be used here:
```suggestion
. Click *New Integration* (*add* icon).
```

[RHSSG link](https://redhat-documentation.github.io/supplementary-style-guide/#user-interface-elements) for reference.

Reply:
I copied this from https://github.com/openshift/openshift-docs/blob/rhacs-docs/modules/amazon-s3-configuring-acs.adoc. Tbh I'm not sure what the purpose is, the button is called `New integration` and has no icon.

Reply:
I removed the entire "(*`add`* icon)" part.